### PR TITLE
[front] - migration: backfill missing editedByUserId

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at providing a migration script to backfill the `editedByUserId` column of the `data_sources` table with the oldest admin of the workspace.

**References:**
 - https://github.com/dust-tt/tasks/issues/1309

## Risk

Touches database

## Deploy Plan

### CODE REVIEW

File: front/migrations/20240912_backfill_editedbyuser_id.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { Workspace } from "@app/lib/models/workspace";
+import { DataSource } from "@app/lib/resources/storage/models/data_source";
+import { MembershipModel } from "@app/lib/resources/storage/models/membership";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript({}, async ({ execute }, logger) => {
+  const dataSources = await DataSource.findAll({
+    where: {
+      editedByUserId: {
+        [Op.is]: undefined,
```
Comment (flvndvd) [inline front/migrations/20240912_backfill_editedbyuser_id.ts:14]:
```suggestion
        [Op.is]: null,
```

File: front/migrations/20240912_backfill_editedbyuser_id.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      },
+      connectorProvider: {
+        [Op.not]: undefined,
```
Comment (flvndvd) [inline front/migrations/20240912_backfill_editedbyuser_id.ts:17]:
```suggestion
        [Op.not]: null,
```

File: front/migrations/20240912_backfill_editedbyuser_id.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+  for (const ds of dataSources) {
+    const workspace = await Workspace.findOne({
+      where: {
+        id: ds.workspaceId,
+      },
+    });
+    assert(workspace, `Failed to find workspace for dataSource ${ds.id}`);
+
+    const oldestAdminMembership = await MembershipModel.findOne({
```
Comment (flvndvd) [inline front/migrations/20240912_backfill_editedbyuser_id.ts:30]: We have `getWorkspaceFirstAdmin`.

**Review by flvndvd — APPROVED** [review]
LGTM once comment addressed

File: front/migrations/20240912_backfill_editedbyuser_id.ts (hunk 1 above)
Comment (JulesBelveze) [reply front/migrations/20240912_backfill_editedbyuser_id.ts:14]:
```
Type { [OpTypes.is]: null; } is not assignable to type
WhereAttributeHashValue<number & {   [CreationAttributeBrand]?: true | undefined; } & {   [ForeignKeyBrand]?: true | undefined; }>
```

**Pull request merged**